Neos.Neos/Classes/View/FusionView.php

```diff
@@ -130,35 +125,6 @@ public function render(): string|ResponseInterface
      */
     protected $securityContext;
 
-    /**
-     * @param string $output
-     * @return string|ResponseInterface If output is a string with a HTTP preamble a ResponseInterface
-     *                                  otherwise the original output.
-     */
-    protected function parsePotentialRawHttpResponse($output)
-    {
-        if ($this->isRawHttpResponse($output)) {
-            return Message::parseResponse($output);
-        }
-
-        return $output;
-    }
```

Comment:
this hack could be removed from the Neos FusionView and the view will use the new Runtime::renderResponse which will handle this.